vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -269,11 +291,7 @@ void vtkF3DImguiActor::RenderFileName()
   winSize.x += 2.f * ImGui::GetStyle().WindowPadding.x;
   winSize.y += 2.f * ImGui::GetStyle().WindowPadding.y;
 
-  // it's super important to set the size of the window manually
-  // otherwise ImGui skip a frame for computing the size resulting in
-  // no UI when doing offscreen rendering
-  ImGui::SetNextWindowSize(winSize);
-  ImGui::SetNextWindowPos(ImVec2(viewport->GetWorkCenter().x - 0.5f * winSize.x, marginTop));
+  ::SetupNextWindow(ImVec2(viewport->GetWorkCenter().x - 0.5f * winSize.x, marginTop), winSize);
```

Comment:
what does window means in this context ?

Reply:
It's the internal window for the widget (basically the black rectangle with rounded corners)